helm/prometheus/values.yaml

```diff
@@ -139,6 +139,9 @@ rulesSelector: {}
 ##
 rules:
   specifiedInValues: true
+  ## What additional rules to be added to the ConfigMap
+  ## You can use this together with `rulesSelector`
+  additionalLabels: {}
```

Comment:
please add a comment on additionalLabels value explaining how to use this variable

Reply:
is this okay?

Reply:
Could you also add in helm/kube-prometheus/Values.yaml ?

Reply:
I forgot helm/prometheus is required in that chart. It should be okay now.